FND/Utilities/Analytics.rdf

```diff
@@ -305,25 +308,24 @@
 		<skos:definition>median of the absolute deviations of observations from the average which may be the arithmetic mean, the median or the mode</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-fnd-utl-alx;NumericIndex">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;Expression"/>
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;ScopedMeasure"/>
+	<owl:Class rdf:about="&fibo-fnd-utl-alx;NumericIndexValue">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-qt-qtu;QuantityValue"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-arr-doc;hasReportingPeriod"/>
-				<owl:onClass rdf:resource="&fibo-fnd-dt-fd;ExplicitDatePeriod"/>
-				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+				<owl:onProperty rdf:resource="&fibo-fnd-utl-alx;isMeasureOf"/>
+				<owl:onClass rdf:resource="&fibo-fnd-utl-alx;ScopedMeasure"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
```

Comment:
Why make this optional? When would this not be set - without the Measure then the value has no context/meaning

Reply:
Still under discussion - it's possible that not all analyses that use scoped measures have an anchor date - maybe they do, and they certainly do for economic indicators, but scoped measure is broader than that.

Reply:
My comment was about isMeasureOf not any date. When would a IndexValue not be linked to a Measure. 
BTW the property name seems wrong now I've looked at it again - it should be isValueOf not isMeasureOf.

Reply:
Still think isValueOf (good name change) should have cardinality of 1.